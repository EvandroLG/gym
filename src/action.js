export function setVideoUrl(url) {
  return {
    type: 'SET_VIDEO_URL',
    url
  };
}

export function setTitleTraining(title) {
  return {
    type: 'SET_TITLE_TRAINING',
    title
  };
}
