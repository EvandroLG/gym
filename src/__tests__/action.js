import { setVideoUrl } from '../action';

describe('action', () => {

  it('should return an expected object', () => {
    const output = setVideoUrl('https://www.youtube.com/watch?v=04p4MWfhpAI');
    expect(output).toMatchSnapshot();
  });
});
