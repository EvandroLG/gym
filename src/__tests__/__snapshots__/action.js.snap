// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action should return an expected object 1`] = `
Object {
  "type": "SET_VIDEO_URL",
  "url": "https://www.youtube.com/watch?v=04p4MWfhpAI",
}
`;
